packages/components/view-switcher/src/view-switcher.spec.tsx

```diff
@@ -1,22 +1,24 @@
 import { useState } from 'react';
```

Comment:
Also wondered why it was js but didn't see it as a concern of my PR :) Thanks for changing it here.